src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TseService.java

```diff
@@ -248,6 +251,45 @@ public String formatApplicationResponses(GenericTseApplicationType application,
                 ).collect(Collectors.joining());
     }
 
+    private String getSingleDecisionMarkdown(TseAdminRecordDecisionType decision, String authToken) {
+        List<GenericTypeItem<DocumentType>> documents = decision.getResponseRequiredDoc();
+        List<String[]> documentsRows = addDocumentRows(documents, authToken);
+        List<String[]> rows = new ArrayList<>(List.of(
+            new String[]{"Notification", decision.getEnterNotificationTitle()},
+            new String[]{"Decision", decision.getDecision()},
+            new String[]{"Decision details", decision.getDecisionDetails()},
+            new String[]{"Date", decision.getDate()},
+            new String[]{"Sent by", TRIBUNAL},
+            new String[]{"Type of decision", decision.getTypeOfDecision()}
+        ));
+        rows.addAll(documentsRows);
+        rows.addAll(List.of(
+            new String[]{"Additional information", decision.getAdditionalInformation()},
+            new String[]{"Decision made by", decision.getDecisionMadeBy()},
+            new String[]{"Name", decision.getDecisionMadeByFullName()},
+            new String[]{"Sent to", decision.getSelectPartyNotify()}
+        ));
+
+        return MarkdownHelper.createTwoColumnTable(new String[]{"Decision", ""}, rows);
+    }
+
+    private String formatApplicationDecision(GenericTseApplicationType application, String authToken) {
+
+        String decisionsMarkdown = "";
+        if (application.getAdminDecision() != null) {
+            // Multiple decisions can be made for the same application but we are only showing the last one for now
```

Comment:
Why are we only showing the last?  This doesn't look accurate given the code below it?

Reply:
Just checked with Boye, he wants just the recent decision. Ran a manual test it only shows most recent

Reply:
Ah I see now, the `.reduce` part is responsible for getting the last item, is this a little convoluted? Would something like this be better?

```
var decisions = application.getAdminDecisions(); 
if (isEmpty(decisions)) {
  return "";
}
var lastItem = decisions.get(decisions,size() - 1);
var markdown = getSingleDecisionMarkdown(lastItem.getValue(), authToken);
```